Resources/config/services.xml

```diff
@@ -19,9 +19,14 @@
     </parameters>
 
     <services>
-        <!-- JWT Encoder / Decoder / Default implementation -->
+        <!--  Default JWT Encoder / Decoder -->
+        <service id="lexik_jwt_authentication.encoder.default" class="Lexik\Bundle\JWTAuthenticationBundle\Encoder\DefaultEncoder">
+            <argument type="service" id="lexik_jwt_authentication.jws_provider"/>
+        </service>
```

Comment:
Just like with the key loaders, I think you can make this private. Then people will be forced to use the alias `lexik_jwt_authentication.encoder`, which is what we want.